### ISSUE TITLE
Update process about database SQL scripts 

### ISSUE THREAD

**Issue by IagoSRL** [loconomics/loconomics#852]
### Long Description
Our current approach for managing upgrades of the database lead to some problems, like not clearly knowing at which point is a database (which scripts were already applied), and where locate previous release scripts (since we remove them after a release, the previous releases tags need to be checked to locate scripts.

### Proposal
- Create release folders under _DBUpdate where place specific scripts; no need to prefix names with the release, just a sorting prefix.
- Keep them forever
- Add a table at database were to specify version of the database/back-end installed.
  - A SQL script needs to do it after apply all the upgrade steps, so a template would be useful
  - or even better: an automated process that executes the upgrade scripts and version update.
  - log the commit hash too, maybe?
  - log every script executed, with result?

### New Nodejs back-end
I have doubts about Loopback here. @ndarilek, Do you know if Loopback is able to automatically apply database changes, based on model definitions? If not, some other process makes more sense with it?

**Comment by IagoSRL** [comment]
@samwa I created this based on our latest conversations, add something you think I forgot or recommendations. Obviously, everything should be more automated, but want to know first about how Loopback manage things in case it solve this for us.

**Comment by ndarilek** [comment]
Yes, LoopBack solves this for us and the solution is already implemented.

**Comment by IagoSRL** [comment]
Thanks for confirming quickly @ndarilek. Please, when you have some time, document how we should use the new back-end to do database changes and get them applied, so we stop doing manual/direct changes to db and get this benefits. Something concise and useful, plus links to any official documentation and limitations (in case there is; thinking on things like renaming, maybe others).

**Comment by IagoSRL** [comment]
We need to update this issue since we wanted to rely on Loopback for database upgrades but we have moved from there to Sailsjs and another ORM, so we should update how will we approach that. @ndarilek feels free to close and move this issue to the loconomics-data repo if makes more sense.
